cachito/paths.py

```diff
@@ -0,0 +1,38 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+import logging
+import shutil
+
+from pathlib import Path
+
+log = logging.getLogger(__name__)
+
+
+class RequestBundleDir(type(Path())):
+    """
+    Represents a directory tree for a request
+
+    :param int request_id: the request ID.
+    :param str root: optional the root directory. If omitted, it defaults to
+        ``cachito_bundles_dir`` from worker config.
+    """
+
+    go_mod_cache_download_part = Path('pkg', 'mod', 'cache', 'download')
+
+    def __new__(cls, request_id, root):
+        self = super().__new__(cls, root, 'temp', str(request_id))
+
+        self.request_id = str(request_id)
```

Comment:
How about making this a private property (i.e. `self._request_id`)?

Reply:
Except constructing the `bundle_archive_path`, `request_id` is not used anywhere else. Hence, I removed `self.request_id`.